corehq/apps/reports/models.py

```diff
@@ -575,6 +576,8 @@ def get_report_content(self, lang, attach_excel=False):
                 },
                 None,
             )
+        except ESError:
+            raise ESError
```

Comment:
I might be missing something, why catch at all if re-raising without doing anything with it?

Reply:
Left this in by mistake, removed in the new PR!